package.json

```diff
@@ -254,7 +254,8 @@
     "@expo/config-plugins/glob": "^10.3.10",
     "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A19.0.0#~/.yarn/patches/@metamask-network-controller-npm-19.0.0-a5e0d1fe14.patch",
     "@solana/web3.js/rpc-websockets": "^8.0.1",
-    "@metamask/network-controller@npm:^19.0.0": "patch:@metamask/network-controller@npm%3A19.0.0#~/.yarn/patches/@metamask-network-controller-npm-19.0.0-a5e0d1fe14.patch"
+    "@metamask/network-controller@npm:^19.0.0": "patch:@metamask/network-controller@npm%3A19.0.0#~/.yarn/patches/@metamask-network-controller-npm-19.0.0-a5e0d1fe14.patch",
+    "@metamask/gas-fee-controller@npm:^15.1.1": "patch:@metamask/gas-fee-controller@npm%3A15.1.2#~/.yarn/patches/@metamask-gas-fee-controller-npm-15.1.2-db4d2976aa.patch"
```

Comment:
Do we need this line? It seems like line 309 is already taking care of the resolution, and I don't see mention of `@metamask/gas-fee-controller` ^15.1.1 in the lockfile.

Reply:
I thought it was needed for transitive dependencies like [this one](https://github.com/MetaMask/metamask-extension/blob/f011d0a761c9df515938b27a875545615bd07769/yarn.lock#L6394).

Reply:
Oh sorry! Yes, I see. Makes sense.